app.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const { getCategories } = require("./controllers/games-controller");
+const app = express();
+
+app.use(express.json());
```

Comment:
Don't need to use this yet - remove for now then add back in in future ticket when it's needed.